### ISSUE TITLE
Can't install on oc v3 from the backend

### ISSUE THREAD

**Issue by chrisvidal** [dead23angel/octovisit#1]
receiving this error

```
- Root composer.json requires cwteam/octovisit-plugin ^1.0, found cwteam/octovisit-plugin[dev-master] but it does not match the constraint.`
```

**Comment by chrisvidal** [comment]
I think you need to create a tag `v1.0` maybe?

**Comment by dead23angel** [comment]
@chrisvidal  Please try again, I created the tag

**Comment by chrisvidal** [comment]
thanks, 
same error, not sure why
<img width="630" alt="image" src="https://github.com/dead23angel/octovisit/assets/2263496/70a65a30-5c0b-4048-9873-42fae5e33175">

**Comment by dead23angel** [comment]
@chrisvidal Please try again

**Comment by chrisvidal** [comment]
Just tried and have now this new error:

Update Failed
"Class "CWTeam\OctoVisit\Updates\Schema" not found" on line 17 of /Users/christophevidal/Sites/oc-courses/plugins/cwteam/octovisit/updates/create_octovisits_table.php

**Comment by chrisvidal** [comment]
you somehow deleted the import `use Schema;` in this class

**Comment by dead23angel** [comment]
Fixed in version 1.0.1, I checked the installation is error-free. Let me know if you still have any errors.

**Comment by chrisvidal** [comment]
still does not work, issue with the 1.0.2

when trying to upgrade to v1.0.2

```
In Plugin.php line 53:
                                           
  A void function must not return a value  
```
